fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraBinaryImpl.java

```diff
@@ -246,7 +248,7 @@ public URI getContentDigest() {
     public String getMimeType() {
         try {
             if (hasProperty(HAS_MIME_TYPE)) {
-                return getProperty(HAS_MIME_TYPE).getString();
+                return getProperty(HAS_MIME_TYPE).getString().replace(FIELD_DELIMITER + XSDstring.getURI(), "");
```

Comment:
I appreciate the explanation, @acoburn. It first pass, it looked like you were just cleaning out the `FIELD_DELIMITER` cruft from the past.
I would like to give this PR a closer look, but have limited time today. If someone in addition to @ajs6f gives it the thumbs-up, feel free to move it forward. Otherwise, I will aim to give it a closer look tonight.